std/prim/char.pol

```diff
@@ -0,0 +1,3 @@
+/// Unicode code points according to definitions D9 and D10 of the
+/// [Unicode standard](https://www.unicode.org/versions/Unicode15.0.0/ch03.pdf#G2212)
+extern Char : Type
```

Comment:
```suggestion
extern Char: Type

```

Reply:
At some point, we should start running `pol fmt` on the examples and the `std` library and check the diff in CI.